### PR TITLE
Also look for the snv_indel_report_clean_unfiltered_file and snv_indel_report_clean_filtered_file by the subject's name

### PR DESCRIPTION
**Pull request by susannasiebert**

### CODE REVIEW

File: lib/perl/Genome/Model/Build/ClinSeq/FileAccessors.pm [hunk 1]
```diff
@@ -353,10 +360,17 @@ sub snv_indel_report_clean_filtered_file {
...
-    $self->warning_message("unable to find " .
-      $snv_indel_report_clean_filtered_file);
-    return;
+  } else{
+    my $snv_indel_report_clean_filtered_file2 = File::Spec->catfile(
+      $snv_indel_report_dir,
+      $self->subject->name . "_final_filtered_clean.tsv");
+    if(-e $snv_indel_report_clean_filtered_file2) {
+      return $snv_indel_report_clean_filtered_file2;
+    }else {
```
Comment (tmooney) [inline lib/perl/Genome/Model/Build/ClinSeq/FileAccessors.pm]: The spacing between the two `else`s in this subroutine is not consistent.

**Comment by susannasiebert** [comment]
I pushed a commit to deduplicate these two subroutines which also cleans up the formatting.

File: lib/perl/Genome/Model/Build/ClinSeq/FileAccessors.pm [hunk 2]
```diff
@@ -326,38 +326,40 @@ sub wgs_exome_snv_summary_dir {
...
+    }
+    else {
+        my $snv_indel_report_clean_file2 = File::Spec->catfile($snv_indel_report_dir, $self->subject->name . "_final_${type}_clean.tsv");
+        if (-e $snv_indel_report_clean_file2) {
+            return $snv_indel_report_clean_file2;
+        }
+        else {
+            $self->warning_message(
+                "unable to find snv_indel_report_clean_${type}_file in location " .
+                "$snv_indel_report_clean_file or  $snv_indel_report_clean_file2"
```
Comment (tmooney) [inline lib/perl/Genome/Model/Build/ClinSeq/FileAccessors.pm]: There are two spaces between `or` and `$snv_indel_report_clean_file2`.

Comment (susannasiebert) [inline lib/perl/Genome/Model/Build/ClinSeq/FileAccessors.pm]: Fixed.

**Pull request merged**